examples/analytics-vendors.amp.html

```diff
@@ -1007,6 +1008,31 @@
 </amp-analytics>
 <!-- End Ibeat Analytics example -->
 
+<!-- Umeng+ Analytics Example -->
+<amp-analytics type="umenganalytics" id="umenganalytics">
+  <script type="application/json">
+    {
+      "vars": {
+        "siteid": "YOUR_SITEID"
```

Comment:
I was testing locally with your config here. I got error `log.js:162 [AmpAnalytics umenganalytics] Analytics config could not be parsed. Is it in a valid JSON format? SyntaxError: Unexpected token  in JSON at position 47`
I check it's because of `_` here. I guess the `_` here is not a standard underscore. Please fix it. 

This is also why I suggest adding default trigger to your vendor config. Because it's just too common to misconfigure the json object 😢. 

Please fix the example here.